PLUGINS.md

```diff
@@ -35,11 +36,11 @@ If the NativeScript framework does not expose a native API that you need, you ca
 
 * The plugin must be a valid npm package.
 * The plugin must expose a built-in native API or a native API available via custom native libraries.
-* The plugin must be written in JavaScript or TypeScript and must comply with the CommonJS specification. If written in TypeScript, make sure to include the compiled `JavaScript` file in your plugin.
+* The plugin must be written in JavaScript or TypeScript and must comply with the CommonJS specification. If written in TypeScript, make sure to include the compiled JavaScript file in your plugin.
```

Comment:
`NOTE: Installing CocoaPods sets the deployment target of your app to iOS 8, if not already set to iOS 8 or later. This change is required because CocoaPods are installed as shared frameworks to ensure that all symbols are available at runtime.`, as @ikoevska's suggested.